### ISSUE TITLE
can't open any folders on Ubuntu 23.04

### ISSUE THREAD

**Issue by nyck33** [getcursor/cursor#1023]
1) If you can, please include a screenshot of your problem

![image](https://github.com/getcursor/cursor/assets/25189545/6aae3733-5203-4e78-aa3e-2e5952eecfe7)

Anything after that is frozen. 

2) Please include the name of your operating system: done
3) If you can, steps to reproduce are super helpful

Install the AppImage, chmod a+x AppImage, then click on the AppImage icon in my Downloads folder.

When first installing had errors on  setting either cursor or code commands.

**Comment by jakobFNF** [comment]
The whole app is frozen?

**Comment by nyck33** [comment]
No, I just can't use the mouse to select any folders, can use arrow keys to navigate and click Enter to go into subdirectories but then can't really launch anything so I have to press ESC to go back to that first "screen" I posted.

**Comment by nyck33** [comment]
This time I couldn't even ESC out so have had to hard reset Ubuntu on three occasions.

**Comment by gebeer** [comment]
Having a similar issue on Arch with dwm window manager. File chooser dialogue can only be navigated with keyboard. Mouse is captured, cannot click anywhere else on my desktop. Only after Esc key pressed.
When trying to get window properties for file chooser, I get:
```
xprop
xprop: error: Can't grab the mouse.
```
